### PR TITLE
[feg] s8 create session response and delete session implementation

### PR DESCRIPTION
**Pull request by uri200**
Signed-off-by: Oriol Batalla <obatalla@fb.com>

## Summary

This PR has:
- Addition of more fields to CreateSessionResponse
- Handling of DeleteSession 
- Handling of SendEcho (with the addition of a new channel to communicate between processes)
- Addition of all commands to proxy relay NH
- Some changes on tools/client to run the proxy either with initial echo or not

## Test Plan

make precommit at feg
## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: feg/gateway/services/s8_proxy/servicers/s8_proxy_test.go [hunk 1]
```diff
@@ -36,7 +37,7 @@ func TestS8Proxy(t *testing.T) {
 		return
 	}
 	defer mockPgw.Close()
-	t.Logf("Running PGW at %s\n", mockPgw.LocalAddr().String())
+	//t.Logf("Running PGW at %s\n", mockPgw.LocalAddr().String())
```
Comment (themarwhal) [inline feg/gateway/services/s8_proxy/servicers/s8_proxy_test.go:40]: remove?

File: feg/protos/s8_proxy.proto [hunk 2]
```diff
@@ -118,31 +117,32 @@ message Fteid {
 }
 
 message CreateSessionResponsePgw {
-    Fteid pgw_fteid_u = 2 ;
     string subscriber_ip = 3;
```
Comment (themarwhal) [inline feg/protos/s8_proxy.proto:120]: It might be helpful to put in comments where these values come from. And 3gpp spec.

**Review by themarwhal — APPROVED** [review]

File: feg/protos/s8_proxy.proto (hunk 2 above)
Comment (uri200) [reply feg/protos/s8_proxy.proto:120]: Hey @themarwhal the 3gpp specs reference are in the top of this file at
http://github.com/magma/magma/blob/master/feg/protos/s8_proxy.proto#L33-L33

Comment (themarwhal) [reply feg/protos/s8_proxy.proto:120]: oooo didn't see!

**Pull request merged**